packages/page-settings/src/Metadata/Extensions.tsx

```diff
@@ -43,32 +43,52 @@ function Extensions ({ chainInfo, className }: Props): React.ReactElement<Props>
     [chainInfo, extensions, selectedIndex, toggleBusy]
   );
 
+  const headerRef = useRef([
+    [t('Extensions'), 'start']
+  ]);
+
   return (
-    <div className={className}>
+    <Table
+      className={className}
+      empty={t<string>('No Upgradable extensions')}
+      header={headerRef.current}
+    >
+
       {extensions
-        ? options.length
+        ? (options.length
           ? (
             <>
-              <Dropdown
-                label={t<string>('upgradable extensions')}
-                onChange={setSelectedIndex}
-                options={options}
-                value={selectedIndex}
-              />
-              <Button.Group>
-                <Button
-                  icon='upload'
-                  isDisabled={isBusy}
-                  label={t<string>('Update metadata')}
-                  onClick={_updateMeta}
-                />
-              </Button.Group>
+              <tr>
+                <td>
+                  <Dropdown
+                    label={t<string>('upgradable extensions')}
+                    onChange={setSelectedIndex}
+                    options={options}
+                    value={selectedIndex}
+                  />
+                </td>
+              </tr>
+
+              <tr>
```

Comment:
```suggestion
              <tr className='isOdd'>
```

Gets rid of the button background.